test/testdata/rewriter/test_case.rb

```diff
@@ -25,23 +17,6 @@ def self.test(*args, &block)
     bar # error: Method `bar` does not exist on `MyTest`
   end
 
-  # Method calls that shouldn't be rewritten
-  tesst "invalid", "method name" do # error: Method `tesst` does not exist on `T.class_of(MyTest)`
-  end
-
-  test "invalid", "parameter count" do
-  end
-
-  test "no block argument"
-
-  test :not_a_string do
-  end
-
-  test :not_a_string do
-    assert true # error: Method `assert` does not exist on `T.class_of(MyTest)`
-  end
```

Comment:
ooc, why are we deleting tests? shouldn't the test pass before and after the changes?

Reply:
Because the rewriter now only rewriters `setup {}` into `def initialize`, some of these tests aren't actually verifying any existing behaviour. I simplified the tests to only verify the scenarios we rewrite:
- If there are invocations to `test "something" {}`, we rewrite the `setup` block into `initialize`
- Otherwise, we do nothing